docker-for-windows/troubleshoot.md

```diff
@@ -286,43 +277,37 @@ script](https://github.com/moby/moby/issues/24388).
 
 ### Virtualization
 
-In order for Docker Desktop for Windows to function properly your machine needs:
+In order for Docker Desktop to function properly your machine needs:
 
 1. [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/hyper-v-technology-overview)
    installed and working
 
 2. Virtualization enabled
 
-
 #### Hyper-V
 
-
-Docker Desktop for Windows requires a Hyper-V as well as the Hyper-V Module for Windows
-Powershell to be installed and enabled. The Docker Desktop for Windows installer enables
+Docker Desktop requires Hyper-V as well as the Hyper-V Module for Windows
+Powershell to be installed and enabled. The Docker Desktop installer enables
 it for you.
 
-Docker Desktop for Windows also needs two CPU hardware features to use Hyper-V: Virtualization and SLAT (Second Level Address Translation), which is also called RVI (Rapid Virtualization Indexing).
-On some systems, Virtualization needs to be enabled in the BIOS. The steps required are vendor-specific, but typically the BIOS option is called `Virtualization Technology (VTx)` or something similar. Run the command `systeminfo` to check all required Hyper-V features. See [Pre-requisites for Hyper-V on Windows 10](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/reference/hyper-v-requirements) for more details.
+Docker Desktop also needs two CPU hardware features to use Hyper-V: Virtualization and SLAT (Second Level Address Translation), which is also called RVI (Rapid Virtualization Indexing). On some systems, Virtualization must be enabled in the BIOS. The steps required are vendor-specific, but typically the BIOS option is called `Virtualization Technology (VTx)` or something similar. Run the command `systeminfo` to check all required Hyper-V features. See [Pre-requisites for Hyper-V on Windows 10](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/reference/hyper-v-requirements) for more details.
```

Comment:
s/SLAT (Second Level Address Translation/Second Level Address Translation (SLAT)

s/RVI (Rapid Virtualization Indexing)/Rapid Virtualization Indexing (RVI)

Reply:
Good catch.

Reply:
Fixed in commit 59aa7e5.